barretenberg/cpp/src/barretenberg/relations/translator_vm/translator_decomposition_relation_2.cpp

```diff
@@ -0,0 +1,20 @@
+#include "barretenberg/relations/translator_vm/translator_decomposition_relation_impl.hpp"
+#include "barretenberg/translator_vm/translator_flavor.hpp"
+namespace bb {
+// Split up due to compile time, used to use DEFINE_SUMCHECK_RELATION_CLASS
+template void TranslatorDecompositionRelationImpl<TranslatorFlavor ::FF>::accumulate<
+    bb ::Relation<TranslatorDecompositionRelationImpl<TranslatorFlavor ::FF>>::SumcheckArrayOfValuesOverSubrelations,
+    TranslatorFlavor ::AllValues>(
```

Comment:
Weird that there are spaces before the scope resolution operators here. That's kosher?

Reply:
its how the macro expanded,  it is 'kosher' in that it compiles, but ill clean this up